### PR TITLE
Fix annotations in file view

### PR DESCRIPTION
**Pull request by LukasReschke**
@icewind1991

### CODE REVIEW

File: lib/private/files/view.php [hunk 1]
```diff
@@ -435,11 +439,18 @@ public function unlink($path) {
 
 	/**
 	 * @param string $directory
+	 * @param bool $empty
```
Comment (LukasReschke) [inline lib/private/files/view.php]: @icewind1991: Where is empty actually used in that method?

**Comment by MorrisJobke** [comment]
Looks good :+1:

File: lib/private/files/view.php (hunk 1 above)
Comment (nickvergessen) [inline lib/private/files/view.php]: Didn't find a call that uses it, so I guess it could be removed.

Comment (nickvergessen) [inline lib/private/files/view.php]: https://github.com/owncloud/core/commit/46a57a9f05eea77889ea33082b5c2259d89acc7a

File: lib/private/files/view.php [hunk 2]
```diff
@@ -435,11 +439,18 @@ public function unlink($path) {
...
+	 * @return bool|mixed
 	 */
 	public function deleteAll($directory, $empty = false) {
 		return $this->rmdir($directory);
 	}
 
+	/**
+	 * @param string $path1
+	 * @param string $path2
+	 * @return bool|mixed
```
Comment (nickvergessen) [inline lib/private/files/view.php]: mixed includes bool already?

File: lib/private/files/view.php [hunk 3]
```diff
@@ -212,6 +212,10 @@ protected function removeMount($mount, $path) {
 		}
 	}
 
+	/**
+	 * @param string $path
+	 * @return bool|mixed
```
Comment (nickvergessen) [inline lib/private/files/view.php]: same here, mixed already includes bool

File: lib/private/files/view.php (hunk 2 above)
Comment (LukasReschke) [inline lib/private/files/view.php]: I actually suspect that the mixed is not going to happen very often but the called functions return null at some point (or possibly other values) – I'd think this makes it cleaner to read.

Comment (nickvergessen) [inline lib/private/files/view.php]: well okay then

File: lib/private/files/view.php (hunk 1 above)
Comment (MorrisJobke) [inline lib/private/files/view.php]: I couldn't find it too.

**Comment by MorrisJobke** [comment]
I dropped the useless parameter.

**Comment by nickvergessen** [comment]
:+1:

**Comment by MorrisJobke** [comment]
I just dropped the parameter in the PHPDoc too.

**Comment by MorrisJobke** [comment]
still :+1: from me

**Pull request merged**